core/src/main/scala/kafka/coordinator/transaction/TransactionMarkerChannelManager.scala

```diff
@@ -71,18 +71,19 @@ object TransactionMarkerChannelManager {
       false,
       new ApiVersions
     )
-    val channel = new TransactionMarkerChannel(config.interBrokerListenerName, metadataCache, networkClient, time)
+    val txnMarkerChannel = new TransactionMarkerChannel(config.interBrokerListenerName, txnStateManager, metadataCache, networkClient, time)
 
-    val sendThread: InterBrokerSendThread = {
+    val txnMarkerSendThread: InterBrokerSendThread = {
       networkClient.wakeup()
```

Comment:
I'm not sure either, it is from the old code. My understanding is that `networkClient` is only pollable from the `sendThread`, and hence `wakeup` is not required. But maybe I'm missing something here, cc @dguy